cross_list_courses/static/cross_list_courses/css/index.css

```diff
@@ -0,0 +1,7 @@
+
+div.row div.col-mid-12 button.center-block {
```

Comment:
it should have been col-md-12. But you are right, it doesn't do anything, getting rid of it and the file